pkg/api/v1/types.go

```diff
@@ -4428,3 +4428,37 @@ type NodeResources struct {
 	// Capacity represents the available resources of a node
 	Capacity ResourceList `protobuf:"bytes,1,rep,name=capacity,casttype=ResourceList,castkey=ResourceName"`
 }
+
+const (
```

Comment:
Not sure if it is worth creating a file. types.go is filled with many constants already.